markdown/chapters/univariate-analysis/main.rmd

```diff
@@ -0,0 +1,1672 @@
+
+Now we consider the shape of the distributions of the responses for the control and truncated plots, with the response set for logarithmically-scaled plot excluded from this, since the high magnitude values will stretch out the x-axis and not allow us to properly see the distributions.
+
+```{r, fig.cap = "Density plot showing distributions of responses regarding the control and truncated plots"}
+
+brks <- c("Control", "Truncated", "Logarithmic")
+vals <- c("#1c9e77", "#d95f02", "#7570b3")
+
+ggplot() +
+  geom_density(data = as.data.frame(control), aes(x=control, col = "Control"))+
+  geom_density(data = as.data.frame(truncated), aes(x=truncated, col = "Truncated"))+
+  labs(x="Response", y="Density")+
+  scale_colour_manual(name = " ", breaks = brks[1:2], values = vals[1:2])+
+  theme_classic()
+
+```
+As expected from the above analysis the two distributions are very similar and we can see that they are, in fact, very much non-normal and asymmetrical. Similarly, the box plot of the response sets (below) displays the similarity between them, where the boxes look almost identical aside from the black square depicting the mean for the truncated plot being slightly higher than for the control.
+
+
+```{r}
+y_scale_1_all <- as.data.frame(y_scale_1_all)
+resp <- c(control, truncated)
+type <- c(rep('Control', 69), rep('Truncated', 70)) 
+stats_1 <- data.frame(resp, type)
+
+ggplot(data = stats_1, aes(x=type, y=resp, fill=type))+
+geom_boxplot(outlier.colour="black", outlier.shape=1,
+             outlier.size=2, notch=F)+
+  theme_classic()+
+  stat_summary(fun=mean, geom="point", shape=15, size=4)+
+  scale_fill_manual(breaks = brks, values = vals)+
+  scale_y_continuous(labels = seq(1, 7, 1), breaks = seq(1, 7, 1))+
+  ylab("Value")+
+  xlab("Plot Type")
+```
+
+Now consider the response distribution for the logarithmically-scaled plot, after removing the two responses of "10^15" and "10^9", as well as the values at 1000.
+
+```{r, fig.cap = "Density plot showing distributions of responses regarding the logarithmic scaled plot, after removing values of greater or equal to 1000"}
+
+
+ggplot() +
+  geom_density(data = as.data.frame(logarithmic), aes(x=logarithmic, col = "Logarithmic"))+
+  labs(x="Response", y="Density")+
+  scale_colour_manual(name = " ", breaks = brks[3], values = vals[3])+
+  theme_classic()
+
+```
+We see that after removing the outlying values, taken as any value above 1000, the distribution is positively skewed and centres around 40 with a tail up to 150. The summary statistics for the outlier removed set shows a mean of 37.45, with a range of $[9, 150]$ and median of 35. The IQR shows that $50\%$ of these values lie in the range $[22.5, 40]$, meaning we appear to potentially have a greater underestimation than for the other two plots, also shown by the mean.
```

Comment:
```suggestion
```
We see that after removing the outlying values, taken as any value above 1000, the distribution is positively skewed and centres around 40 with a tail up to 150. The summary statistics for the outlier removed shows a mean of 37.45, with a range of $[9, 150]$ and median of 35. The IQR shows that $50\%$ of these values lie in the range $[22.5, 40]$, meaning we appear to potentially have a greater underestimation than for the other two plots, also shown by the mean.
```